relayer-cli/src/commands/tx/client.rs

```diff
@@ -101,3 +102,52 @@ impl Runnable for TxUpdateClientCmd {
         }
     }
 }
+
+#[derive(Clone, Command, Debug, Options)]
+pub struct TxUpgradeClientCmd {
+    #[options(free, required, help = "identifier of the destination chain")]
+    dst_chain_id: ChainId,
+
+    #[options(
+        free,
+        required,
+        help = "identifier of the chain which underwent upgrade (source chain)"
+    )]
+    src_chain_id: ChainId,
+
+    #[options(
+        free,
+        required,
+        help = "identifier of the client to be upgraded on destination chain"
+    )]
+    dst_client_id: ClientId,
```

Comment:
I think we should make `dst_client_id` optional and if not entered to upgrade all clients of `src_chain_id`  present on `dst_chain_id`. The way it is, with a lot of clients, the user needs to query all clients and issue the upgrade for each client. 
Also, we should have a `hermes client upgrade..` command in place of or in addition to this command.

Reply:
Do you want to do this in this PR or a follow-up one? If the latter, then we should open an issue to track this.

Reply:
Follow up PR, thanks!

Reply:
https://github.com/informalsystems/ibc-rs/issues/763